notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/INotification.java

```diff
@@ -0,0 +1,26 @@
+package com.microsoft.windowsazure.messaging.notificationhubs;
+
+import java.util.Map;
+
+/**
+ * Wraps the notifications that may be delivered via any channel.
+ */
+public interface INotification {

```

Comment:
Java doesn't typically use the "I" prefix on interfaces, right?

Should we keep this named `NotificationMessage` and change the name of the current class implementing it?

Reply:
Agreed, we should have `NotificationMessage` as the interface and something like `FirebaseNotificationMessage` or whatever we want to call it as the implementation

Reply:
Fixed in de6ef36cd9fcc336e182ad3b7e1fd64432329c90